pennylane/wires.py

```diff
@@ -123,6 +123,23 @@ def tolist(self):
         """
         return list(self.wire_tuple)
 
+    def at_index(self, idx):
+        """Returns the wire at index ``idx``.
+
+        >>> w = Wires([0, 'q1', 16])
+        >>> w[1]
+        'q1'
+        >>> w[2]
```

Comment:
Should this be `w.at_index(2)`? Same for `w[1]`.

Reply:
AAAh, sorry!

Reply:
```suggestion
        >>> w.at_index(2)
```